docs/4.6. Validating Access Tokens.md

```diff
@@ -4,7 +4,7 @@ _(c) AMWA 2021, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 There are a number of steps that need to be considered while [validating the access token](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#validation-of-access-token).
 
 - Check that the JWT is well-formed, such as JWT contains three segments, Header, Payload and Signature, separated by period ('.') characters. Each segment is Base64url encoded.
-- Verify the Signature, by generating a new Base64url-encoded Signature using the issuer public key and verify that it matches the original Signature, to ensure the token has not been tampered with. In event the issuer [public key](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, it should attempt to obtain the missing public key via the the token `iss` claim as specified in [RFC 8414](https://tools.ietf.org/html/rfc8414 "OAuth 2.0 Authorization Server Metadata") section 3, and temporarily respond with an HTTP 503 code in order to avoid blocking the incoming authorized request.
+- Verify the Signature, by generating a new Base64url-encoded Signature using the issuer's public key and verify that it matches the original Signature, to ensure the token has not been tampered with. In event the issuer [public key](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, it should attempt to obtain the missing public key via the the token `iss` claim as specified in [RFC 8414](https://tools.ietf.org/html/rfc8414 "OAuth 2.0 Authorization Server Metadata") section 3, and temporarily respond with an HTTP 503 code in order to avoid blocking the incoming authorized request.
```

Comment:
> In event the issuer [public key](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) is not known, it should attempt to obtain the missing public key via the the token `iss` claim as specified in [RFC 8414]

I believe this section is assuming that the JWKs is cached by the client but I am a little unsure why it's "not known"

Reply:
One for @simonlo-sony to answer I think